analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantModifier.CSharp11.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+public class Foo
+{
+    public static void Method()
+    {
+        checked // Noncompliant, FP, nint/IntPtr is not considered an integral number type
```

Comment:
The comment is misleading. `nint`/`nuint` are considered integral number types: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/integral-numeric-types

Reply:
You are right, I was just refering to the `KnownType.IntegralNumbers` collection of types that is used by this rule, but I should have phrased it better. 

I will refactor this to something like `is not part of the KnownType.IntegralNumbers collection` instead.